.gitignore

```diff
@@ -9,6 +9,7 @@ META.*
 MYMETA.*
 Makefile.PL
 *.gz
+!t/data/engine/**/reference-output/**/*.gz
```

Comment:
@dekarl right, thanks. It's been a while since I touched this code. But I would be happy to change gzip to something else if gzip still produces some inconsistent results on certain platforms and thus causes tests to fail.